src/main/resources/data-submitter-schema_v1.json

```diff
@@ -0,0 +1,386 @@
+    },
+    "alternativeDataSharingPlanFileName": {
+      "type": "string",
+      "label": "Upload your alternative sharing plan (file upload)",
+      "description": "Upload your alternative sharing plan (file upload)"
+    },
+    "alternativeDataSharingPlanDataSubmitted": {
+      "type": "array",
+      "label": "Data will be submitted",
+      "description": "Upload your alternative sharing plan (file upload)",
+      "items": {
+        "type": "string",
+        "enum": [
+          "Within 3 months of the last data generated or last clinical visit",
+          "By batches over Study Timeline (e.g. based on clinical trial enrollment benchmarks)"
+        ]
+      }
+    },
+    "alternativeDataSharingPlanDataReleased": {
+      "type": "boolean",
+      "label": "Data to be released will meet the timeframes specified in the NHGRI Guidance for Data Submission and Data Release",
+      "description": "Data to be released will meet the timeframes specified in the NHGRI Guidance for Data Submission and Data Release"
+    },
+    "alternativeDataSharingPlanTargetDeliveryDate": {
+      "type": "string",
+      "format": "date",
+      "label": "Target Delivery Date",
+      "description": "Target Delivery Date"
+    },
+    "alternativeDataSharingPlanTargetPublicReleaseDate": {
+      "type": "string",
+      "format": "date",
+      "label": "Target Public Release Date",
+      "description": "Target Public Release Date"
+    },
+    "alternativeDataSharingPlanControlledOpenAccess": {
+      "type": "string",
+      "label": "Does the data need to be managed under Controlled or Open Access?",
+      "description": "Does the data need to be managed under Controlled or Open Access?",
+      "enum": ["Controlled Access", "Open Access"]
+    },
+    "dataAccessCommitteeId": {
+      "type": "number",
+      "label": "Please select which DAC should govern requests for this dataset",
+      "description": "Data Access Committee ID"
+    },
+    "consentGroups": {
+      "type": "array",
+      "label": "Consent Groups",
+      "description": "Consent Groups",
+      "items": { "$ref":  "#/$defs/consentGroup"}
+    }
+  },
+  "$defs": {
+    "fileTypeObject": {
+      "required": ["fileType", "functionalEquivalence", "numberOfParticipants"],
+      "type": "object",
+      "properties": {
+        "fileType": {
+          "type": "string",
+          "description": "File Type",
+          "enum": ["Arrays", "Genome", "Exome", "Survey", "Phenotype"]
+        },
+        "functionalEquivalence": {
+          "type": "string",
+          "description": "Functional Equivalence"
+        },
+        "numberOfParticipants": {
+          "type": "number",
+          "description": "# of Participants"
+        }
+      }
+    },
+    "consentGroup": {
+      "required": ["consentGroupName"],
+      "oneOf": [
+        {"required":  ["generalResearchUse"]},
+        {"required":  ["hmb"]},
+        {"required":  ["diseaseSpecificUse"]}
```

Comment:
does this also need to have `poa` as an option in the `oneOf`?

Reply:
`poa` and `otherPrimary` are completely optional. There has to be at most one of these three primary use restrictions, and all others are secondary and optional.